### PR TITLE
Item Sheet V2: Activities tab

### PR DESCRIPTION
**Pull request by Fyorl**
image

There's room for some supplementary information on these activity rows, so the design may change a bit, but for now I haven't thought much about what would go there.

### CODE REVIEW

File: module/documents/item.mjs [hunk 1]
```diff
@@ -90,10 +90,17 @@ export default class Item5e extends SystemDocumentMixin(Item) {
      */
     if ( options.pack || options.parent?.pack ) Hooks.callAll("dnd5e.initializeItemSource", this, data, options);
 
-    ActivitiesTemplate.initializeActivities(data);
```
Comment (Fyorl) [inline module/documents/item.mjs:93]: I noticed there are some times when `_initializeSource` might receive partial source data, such as when creating a temporary Document, i.e. `new Actor({ name: 'foo', type: 'npc' });`. I moved this to `fromSource` instead, which generally expects to instantiate a Document from full source data. It's not foolproof, but it's the best I can think of right now.

File: lang/en.json [hunk 2]
```diff
@@ -180,6 +180,8 @@
 },
 
 "DND5E.ACTIVITY": {
+  "Create": "Create Activity",
+  "Delete": "Delete Activity",
```
Comment (arbron) [inline lang/en.json:184]: For some other localizations recently I've been putting these in an an Action object:

```suggestion
  "Action": {
    "Create": "Create Activity",
    "Delete": "Delete Activity"
  },
```

File: templates/items/parts/item-activities.hbs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<div class="tab activities" data-group="primary" data-tab="activities">
+    {{#each system.activities}}
+    <div class="activity card" data-id="{{ _id }}">
+
+        {{!-- TODO: Add drag handle for re-ordering --}}
+
+        {{!-- Icon --}}
+        <div class="icon">
+            <dnd5e-icon src="{{ img }}"></dnd5e-icon>
```
Comment (arbron) [inline templates/items/parts/item-activities.hbs:9]: You can give activities custom icons, so we'll probably want to handle that similar to the advancement icons.

File: module/applications/item/item-sheet-2.mjs [hunk 4]
```diff
@@ -92,6 +92,55 @@ export default class ItemSheet5e2 extends ItemSheetV2Mixin(ItemSheet5e) {
         }));
       });
     }
+
+    if ( this.isEditable ) {
+      html.find(".activities .activity .name").on("click", this._onEditActivity.bind(this));
```
Comment (arbron) [inline module/applications/item/item-sheet-2.mjs:97]: It would probably still be good to be able to open the activity sheet even if the item is locked.

**Review by arbron — CHANGES_REQUESTED** [review]

File: module/documents/item.mjs (hunk 1 above)
Comment (arbron) [reply module/documents/item.mjs:93]: Seems to be working, but I'll keep an eye out to see if anything funky happens.

**Review by arbron — APPROVED** [review]

**Pull request merged**